package.json

```diff
@@ -99,6 +99,7 @@
     "json-schema": "^0.4.0",
     "simple-get": "^4.0.1",
     "web3-provider-engine/eth-json-rpc-filters": "^5.0.0",
+    "http-cache-semantics": "^4.1.1",
```

Comment:
This is an in-range bump, so we don't need to leave a resolution here for this
```suggestion
```

Reply:
To clarify, this is a good way of bumping an indirect dependency, this is what I would do as well. Just we can remove it again after the lockfile has been updated if it's compatible with the requested ranges. This makes future updates less complicated, as conflicts with these resolutions could cause issues that would be hard to spot.